.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+
+# Cache pip downloads
+cache:
+    # Apparently if you override the install command that silently disables the
+    # cache: pip support. This is less than ideal and @dstufft opened up
+    # travis-ci/travis-ci#3239 to hopefully get that addressed. For now we
+    # manually add the pip cache directory to the build cache.
+    # (from django-oscar/.travis.yml)
+    directories:
+      - ~/.cache/pip
```

Comment:
Fancy! But I'm not sure if it works without the new container based infrastructure: https://github.com/django-oscar/django-oscar/blob/master/.travis.yml#L1

Frankly, this is such a simple project, I wouldn't bother with either.